### PR TITLE
feat: stream ChaosLabs payment

### PR DESCRIPTION
**Pull request by Rozengarden**

### CODE REVIEW

File: src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/ChaosLabsAaveRiskManagementServiceRenewal.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+If this proposal is approved, a stream of the allocated budget will be activated, with a Chaos Labs-controlled account (`0xbC540e0729B732fb14afA240aA5A047aE9ba7dF0`) as the recipient.
+
+Regarding technical implementation, the AIP will call the `createStream()` method of the `IAaveEcosystemReserveController` interface to create a stream of 1,000,000 aUSDT and 1,000,000 GHO for a 12-month duration.
+
+## References
+
+- Implementation: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal_20241012.sol)
+- Tests: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal_20241012.t.sol)
+- [Snapshot](TODO)
```
Comment (brotherlymite) [inline src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/ChaosLabsAaveRiskManagementServiceRenewal.md:149]:
```suggestion
- Snapshot: Direct to AIP
```

File: src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal_20241012.sol [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/src/interfaces/IProposalGenericExecutor.sol';
+import {CollectorUtils} from 'aave-helpers/src/CollectorUtils.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+
+/**
+ * @title Chaos Labs <> Aave Risk Management Service Renewal
+ * @author Aave Chan Initiative
+ * - Snapshot: TODO
```
Comment (brotherlymite) [inline src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal_20241012.sol:12]:
```suggestion
 * - Snapshot: Direct To AIP
```

File: src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal_20241012.sol [hunk 3]
```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
```
Comment (brotherlymite) [inline src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal_20241012.sol:4]:
```suggestion
```

File: src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal_20241012.sol [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        duration: 365 days
+      })
+    );
+
+    CollectorUtils.stream(
+      AaveV3Ethereum.COLLECTOR,
+      CollectorUtils.CreateStreamInput({
+        underlying: AaveV3EthereumAssets.USDC_A_TOKEN,
+        receiver: CHAOS_LABS_RECEIVER,
+        amount: 1_000_000 ether,
```
Comment (brotherlymite) [inline src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal_20241012.sol:37]: @Rozengarden aUSDC has 6 decimals so we should not use ether here.

File: src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal_20241012.t.sol [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 20952046);
+    proposal = new AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal_20241012();
+  }
+
+  /**
+   * @dev executes the generic test suite including e2e and config snapshots
+   */
+  function test_defaultProposalExecution() public {
```
Comment (brotherlymite) [inline src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal_20241012.t.sol:28]: @Rozengarden I would also add that test that we can withdraw the whole amount after 1 year

File: src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/ChaosLabsAaveRiskManagementServiceRenewal.md (hunk 1 above)
Comment (Rozengarden) [reply src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/ChaosLabsAaveRiskManagementServiceRenewal.md:149]: Snapshot will be posted in a few hours

**Review by ianflexa — APPROVED** [review]

File: src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/ChaosLabsAaveRiskManagementServiceRenewal.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+### **Risk Monitoring and Alerting**
+
+We offer real-time monitoring and alerting services through the [Chaos Labs Risk Monitoring Platform](https://community.chaoslabs.xyz/aave/risk/overview). These services allow users to assess ecosystem risk and protocol health at granular levels using dashboards and data visualizations. Users can drill into specific scenarios in which the protocol could be negatively impacted and receive alerts concerning significant changes to on-chain activities that could pose a risk or detriment to the protocol’s health.
+
+As part of our commitment to delivering a world-class risk management solution for the DeFi ecosystem, we constantly iterate and improve the platform with additional features and enhancements. The Aave community will continue to gain access to the newest feature additions and enhancements continuously, allowing users to stay ahead of the curve and make informed decisions while optimizing their overall portfolio performance.
+
+![Screenshot 2024-09-27 at 15.50.34|1408x1710](upload://Aqu3IuxgoK2aEjL54oW9WkwpmnG)
```
Comment (ianflexa) [inline src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/ChaosLabsAaveRiskManagementServiceRenewal.md:83]:
```suggestion

```

Comment (ianflexa) [inline src/20241012_AaveV3Ethereum_ChaosLabsAaveRiskManagementServiceRenewal/ChaosLabsAaveRiskManagementServiceRenewal.md:83]:
```suggestion

```

**Review by ianflexa — CHANGES_REQUESTED** [review]

**Review by ianflexa — DISMISSED** [review]

**Comment by brotherlymite** [comment]
@Rozengarden could you fix lint as CI is failing otherwise

**Review by brotherlymite — APPROVED** [review]

**Pull request merged**